src/api/status/src/views/partials/header.hbs

```diff
@@ -0,0 +1,24 @@
+<head>
+  <meta charset="utf-8" />
+  <base href="/v1/status/" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <link rel="apple-touch-icon" sizes="76x76" href="assets/img/apple-icon.png" />
+  <link rel="icon" type="image/svg+xml" href="./logo.svg" />
+  {{#if active_build_log}}
```

Comment:
Seem like this is the only way to have different `head` for different pages. I tried [this implementation](https://cloudfour.com/thinks/the-hidden-power-of-handlebars-partials/) but no luck, probably only compatible with the handlebars CLI.

Reply:
We can have different heads for specific page, but this was the easier way since it's only 1 HTML line